Python/test-demofuzz.sh

```diff
@@ -4,11 +4,28 @@ set -e -o pipefail
 . venv/bin/activate
 set -u
 
-readonly config='Config.json'
-readonly job_path='../SampleFuzzingJobs/Demofuzz'
-readonly job_params='JobParams.json'
+readonly windows_config='Config.json'
```

Comment:
My suggestion for this file was to split it into two files: one for each platform, so they can be tested independently.

Reply:
Done.